Shoko.Server/API/v3/Controllers/SeriesController.cs

```diff
@@ -134,6 +134,26 @@ public ActionResult<List<SeriesRelation>> GetShokoRelationsBySeriesID([FromRoute
                 .ToList();
         }
         
+        /// <summary>
+        /// Get a paginated list of <see cref="Series"/> without local files, available to the current <see cref="User"/>.
+        /// </summary>
+        /// <param name="pageSize">The page size.</param>
+        /// <param name="page">The page index.</param>
+        /// <returns></returns>
+        [HttpGet("WithoutFiles")]

```

Comment:
Should this be `/Empty` instead? 🤔

Reply:
Nah. `WithoutFiles` signifies better what it returns imo.